src/uwtools/resources/rocoto.jsonschema

```diff
@@ -0,0 +1,69 @@
+{
+    "$defs": {
+        "dateTimePattern": {
+            "type": "string",
+            "pattern": "^(\\d{12}) (\\d{12}) (\\d{2}):(\\d{2}):(\\d{2}):(\\d{2})$"
+        }
+    },
+    "type": "object",
+    "properties": {
+        "workflow": {
+            "type": "object",
+            "properties": {
+                "attrs": {
+                    "type": "object",
+                    "properties": {
+                        "realtime": {
+                            "type": "boolean"
+                        },
+                        "scheduler": {
+                            "type": "string",
+                            "enum": [
+                                "sge",
+                                "lsf",
+                                "lsfcray",
+                                "ll",
+                                "torque",
+                                "moabtorque",
+                                "moab",
+                                "pbspro",
+                                "slurm",
+                                "cobalt"
+                            ]
+                        },
+                        "cyclethrottle": {
+                            "type": "integer",
+                            "minimum": 0
+                        },
+                        "taskthrottle": {
+                            "type": "integer",
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["realtime", "scheduler"]
+                },
+                "cycledefs": {
+                    "type": "object",
+                    "properties": {
+                        "howdy": {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "$ref": "#/$defs/dateTimePattern"
```

Comment:
You shouldn't need to type here since the $ref is handling that - do we need that 'howdy' section though?